test/threads.jl

```diff
@@ -475,7 +475,12 @@ test_nested_loops()
                      append=false))
         out = readchomp(joinpath(dir, "out.txt"))
         err = readchomp(joinpath(dir, "err.txt"))
-        @test contains(out, "libat_store") || contains(out, "atomic_store")
+
+        if Sys.ARCH === :aarch64 || Sys.ARCH === :powerpc64le || Sys.ARCH === :ppc64le
+            @test_broken contains(out, "libat_store") || contains(out, "atomic_store")
```

Comment:
It would probably be good to add a comment that explains why this is marked as broken